### PR TITLE
Add example for dynamic clusters

### PR DESCRIPTION
**Pull request by ahocevar**
This pull request adds an example showing functionality similar to Leaflet's [Leaflet.markercluster](http://leaflet.github.io/Leaflet.markercluster/) plugin.

### CODE REVIEW

**Comment by ahocevar** [comment]
https://deploy-preview-13158--ol-site.netlify.app/examples/clusters-dynamic.html is the new example.

**Comment by tschaub** [comment]
clusters

It looks like the convex hulls render under the cluster markers.  Just want to confirm this is what it is supposed to look like.

File: examples/clusters-dynamic.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+layout: example.html
+title: Dynamic clusters
+shortdesc: Clusters with zoom-to-cluster, hull view, and uncluttering of overlapping features.
+docs: >
+  <p>This example shows open data of subsidized photovoltaic installations in Vienna. Different style functions
+    are used for cluster display, single feature display, convex hull of a cluster, and an uncluttered view of
+    overlapping features. Hovering over a cluster shows its convex hull. Clicking on a cluster zooms to the
+    extent of the contained features. Clicking on a cluster consisting of features that are very close to each other reveals an uncluttered view
```
Comment (tschaub) [inline examples/clusters-dynamic.html:9]: Did you mean "unclustered" instead of "uncluttered" (above as well)?

**Comment by tschaub** [comment]
> It looks like the convex hulls render under the cluster markers. Just want to confirm this is what it is supposed to look like.

I see the same on the https://leaflet.github.io/Leaflet.markercluster/example/marker-clustering-realworld.388.html example.  And I see it looks nicer when zoomed out more than in my example above.

File: examples/clusters-dynamic.html (hunk 1 above)
Comment (ahocevar) [reply examples/clusters-dynamic.html:9]: Not really "unclustered". It's a view for features that share the same location (within one pixel), or are still clustered at the view's `maxZoom`. How would you describe that?

Untitled

The convex hulls are supposed to be rendered below the cluster circle.

**Comment by ahocevar** [comment]
I made the cluster circles a bit transparent, so the convex hulls shine through.

File: examples/clusters-dynamic.html (hunk 1 above)
Comment (tschaub) [reply examples/clusters-dynamic.html:9]: Ah, cool.  How about an “expanded” view?

Comment (ahocevar) [reply examples/clusters-dynamic.html]: Good suggestion, thanks! I changed all occurrences to "expanded".

**Review by tschaub — APPROVED** [review]

**Comment by tschaub** [comment]
Nice example.

**Comment by jipexu** [comment]
👍 so good ! thanks

**Pull request merged**